students/DanCornutt/lessons/lesson09/assignment/database.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+import os
+import argparse
+
+
+# 1 use decorator similar to class example, use argparse as well. logging.disabled = true
+# 2 add context for connecting to mongoDB
+# 3 Recursive function looks at files in local, returns .jpg via os.path, calls itself on all folders in local.
+
+FILE_CONTENTS = []
+
+
+def selective_log():
+    """Item #1 from hw. Add decorators to introduce conditional logging. --In work
+    """
+    pass
+
+
+def context_manager():  # 2
+    """Lesson 5 already uses context manager for MongoDB connection. --In work
+    """
+    pass
```

Comment:
Not sure what the point of those two functions is, but recursive function looks pretty good. Would like to see the requirement to initialize a list outside of the function gone. Should be able to do this by setting a second input parameter key word to initialize to None, and then create the list if it is None.